### PR TITLE
Modified hough-circle for color circle

### PR DESCRIPTION
**Pull request by iory**
hough_circle

### CODE REVIEW

**Comment by k-okada** [comment]
I think there are two cases for color or gray for 'frame' image
https://github.com/iory/opencv_apps/blob/1ffa870096a36d4e24d82a4c1a62e7e465d9d0d2/src/nodelet/hough_circles_nodelet.cpp#L146

**Comment by iory** [comment]
in gray scale, we should output gray scale image?

**Comment by iory** [comment]
https://github.com/iory/opencv_apps/blob/1ffa870096a36d4e24d82a4c1a62e7e465d9d0d2/src/nodelet/hough_circles_nodelet.cpp#L213-L215
This code is meaningless

File: src/nodelet/hough_circles_nodelet.cpp [hunk 1]
```diff
@@ -201,15 +201,18 @@ class HoughCirclesNodelet : public opencv_apps::Nodelet
                         min_circle_radius_,
                         max_circle_radius_ );
 
+      cv::Mat out_image;
+      cv::cvtColor( frame, out_image, cv::COLOR_GRAY2BGR);
```
Comment (k-okada) [inline src/nodelet/hough_circles_nodelet.cpp]: `frame` can be both color and gray

**Comment by k-okada** [comment]
this is conflict, please rebase

**Comment by iory** [comment]
rebased

**Comment by k-okada** [comment]
thx

**Pull request merged**